### DISCUSSION TITLE
New update has gotten rid of all of my beatmaps... again...

### DISCUSSION THREAD

**Discussion by Tremayah** [ppy/osu#25486] [Q&A]
Hi, this is the third time this has happened to me and the second time I am posting about it.
(https://github.com/ppy/osu/discussions/24816).
A very recent update has apparently changed the way osu reads map from the data base, and now all my maps are gone! (again ;-;).
This was fixed last time by I think ppy manually updating my maps folder? Also lots of renaming client realm files. I know nothing about this stuff pls help >.<
![image](https://github.com/ppy/osu/assets/125309136/4e74cc7e-c779-4a8a-953d-021b0f5de723)

**Comment by peppy** [comment]
are you running unofficial or self built versions?

**Reply by Tremayah** [reply]
Not that I know of, no.

**Reply by Tremayah** [reply]
Oh this might look different because of the last time the maps disappeared, there was a bit of save-surgery to try and get them back.

**Reply by peppy** [reply]
Your previous database was renamed to `client_newer_version.realm` because the game detected that an older version was run after updating. Maybe you accidentally did this. You should make sure you only have one copy of the game installed to avoid this.

To fix the issue, you can delete `client.realm` and rename `client_newer_version.realm` to `client.realm`.

**Reply by Tremayah** [reply]
Done, I now get this error message when opening osu 
![image](https://github.com/ppy/osu/assets/125309136/f6f5f132-a07d-47ac-87eb-96426e2a8184)
No maps are present yet.
![image](https://github.com/ppy/osu/assets/125309136/bce779e9-8909-43e0-8247-e3c82f1453cb)
(accidentally opened client.realm in notepad not knowing it would get rid of all of the .realm bits of the file name, osu seems to read "client" fine though. When renamed to literally "client.realm" a new file named "client" is created on game launch.

**Reply by peppy** [reply]
I have no idea. This can't happen unless you've done something weird.

**Reply by Tremayah** [reply]
Almost certainly has something to do with the first time I tried to fix this. That's why the files look a bit different probably.
